core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/enclave_plan_capabilities/enclave_plan_capabilites_builder.go

```diff
@@ -0,0 +1,59 @@
+package enclave_plan_capabilities
+
+import (
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/service"
+)
+
+type EnclavePlanCapabilitiesBuilder struct {
```

Comment:
See my other comments, at first glance this seems like a big bag of everything. We should make it representative of what an instruction can affect IMO (i.e. one or multiple services, one or multiple files)